tests/integration/test_sqs.py

```diff
@@ -1832,6 +1832,56 @@ def receive_message(attribute_names, message_attribute_names=None):
         response = receive_message(["SenderId", "SequenceNumber"])
         assert snapshot.match("multiple_attributes", response)
 
+    @pytest.mark.aws_validated
+    def test_change_visibility_on_deleted_message_raises_invalid_parameter_value(
+        self, sqs_client, sqs_queue
+    ):
+        # prepare the fixture
+        sqs_client.send_message(QueueUrl=sqs_queue, MessageBody="foo")
+        response = sqs_client.receive_message(QueueUrl=sqs_queue, WaitTimeSeconds=5)
+        handle = response["Messages"][0]["ReceiptHandle"]
+
+        # check that it works as expected
+        sqs_client.change_message_visibility(
+            QueueUrl=sqs_queue, ReceiptHandle=handle, VisibilityTimeout=5
+        )
```

Comment:
Using literal "5" for both is a bit confusing, because it looks as these need to be equal. But if I understand it correctly they have nothing to do with each other, so using for example WaitTimeSeconds=3 instead should work as well. And if they need to be equal we should probably store the value in a common variable.

Reply:
good point :-)